OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs

```diff
@@ -438,9 +438,9 @@ public static void SetupSpawnWidget(Widget parent, Session.Slot s, Session.Clien
 		public static void SetupEditableReadyWidget(Widget parent, Session.Slot s, Session.Client c, OrderManager orderManager, MapPreview map)
 		{
 			var status = parent.Get<CheckboxWidget>("STATUS_CHECKBOX");
-			status.IsChecked = () => orderManager.LocalClient.IsReady || c.Bot != null;
+			status.IsChecked = () => orderManager.LocalClient.IsReady;
 			status.IsVisible = () => true;
```

Comment:
Alright, I'm fine with this then. I have yet to do a code review and test, however.

On 2015-07-12, at 10:34 PM, Deniz <notifications@github.com> wrote:

> In OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs:
> 
> > @@ -438,9 +438,9 @@ public static void SetupSpawnWidget(Widget parent, Session.Slot s, Session.Clien
> >  		public static void SetupEditableReadyWidget(Widget parent, Session.Slot s, Session.Client c, OrderManager orderManager, MapPreview map)
> >  		{
> >  			var status = parent.Get<CheckboxWidget>("STATUS_CHECKBOX");
> > -			status.IsChecked = () => orderManager.LocalClient.IsReady || c.Bot != null;
> > +			status.IsChecked = () => orderManager.LocalClient.IsReady;
> >  			status.IsVisible = () => true;
> I was going to do the start countdown too but decided to break it up into smaller PRs so they can be reviewed separately.
> 
> ―
> Reply to this email directly or view it on GitHub.
>